packages/expo-device/package.json

```diff
@@ -0,0 +1,33 @@
+{
+  "name": "expo-device",
+  "version": "1.0.0",
+  "description": "ExpoDevice standalone module",
+  "main": "build/Device.js",
+  "types": "build/Device.d.ts",
+  "scripts": {
+    "build": "expo-module build"
+  },
+  "keywords": [
+    "react-native",
+    "expo",
+    "expo-device"
+  ],
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/expo/expo.git"
+  },
+  "bugs": {
+    "url": "https://github.com/expo/expo/issues"
+  },
+  "author": "650 Industries, Inc.",
+  "license": "MIT",
+  "homepage": "https://github.com/expo/expo/tree/master/packages/expo-device",
+  "peerDependencies": {
```

Comment:
I think right now the convention is to use 

https://github.com/expo/expo/blob/3ff22c99005ff476fc50b6dd91f7697a02234df2/packages/expo-constants/package.json#L34-L37